src/test/java/net/snowflake/client/jdbc/structuredtypes/ResultSetStructuredTypesLatestIT.java

```diff
@@ -39,81 +38,67 @@
 import net.snowflake.client.jdbc.structuredtypes.sqldata.NullableFieldsSqlData;
 import net.snowflake.client.jdbc.structuredtypes.sqldata.SimpleClass;
 import net.snowflake.client.jdbc.structuredtypes.sqldata.StringClass;
-import org.junit.After;
+import net.snowflake.client.providers.FormatProvider;
 import org.junit.Assume;
```

Comment:
what about Assume usage? should we replace it with junit 5 equivalent?

Reply:
Indeed we should, good catch